.github/scripts/bot-pr-gfi-review-triage.sh

```
@@ -0,0 +1,81 @@
+fi
+
+if [ "$is_dry_run" = true ]; then
+  echo "DRY RUN: would post comment to PR #$PR_NUMBER with body:" 
+  echo "---"
+  echo -e "$BODY_TEXT"
+  echo "---"
+  exit 0
+fi
+
+echo "Posting triage request comment to PR #$PR_NUMBER"
+resp=$(curl -fS -X POST -H "Authorization: Bearer $TOKEN" -H "Accept: application/vnd.github+json" -H "User-Agent: hiero-sdk-bot" \
```

Shellcheck (0.11.0)

[warning] resp appears unused. Verify use (or export if used externally).

(SC2034)